src/uattributes/uattributesvalidator.rs

```diff
@@ -115,16 +114,10 @@ pub trait UAttributesValidator {
 
     /// Verifies that the message has a valid time-to-live.
     ///
-    /// # Errors
-    ///
-    /// This default implementation returns an error if [`UAttributes::ttl`] (time-to-live) contains a value less than 0.
-    fn validate_ttl(&self, attributes: &UAttributes) -> Result<(), UAttributesError> {
-        match attributes.ttl {
-            Some(t) if t < 0 => Err(UAttributesError::validation_error(format!(
-                "TTL must not be negative [{t}]"
-            ))),
-            _ => Ok(()),
-        }
+    /// This default implementation always returns Ok(), as u32 ttl can never be < 0
+    fn validate_ttl(&self, _attributes: &UAttributes) -> Result<(), UAttributesError> {
```

Comment:
if there is nothing to validate by default, then I would propose to remove the function from the trait and instead add it to the RequestValidator only.

Reply:
done